src/dynamics/se/dyn_comp.F90

```diff
@@ -9,7 +9,8 @@ module dyn_comp
 use cam_constituents,       only: const_get_index, const_is_wet, const_qmin
 use cam_constituents,       only: readtrace
 use air_composition,        only: const_is_water_species
-use cam_control_mod,        only: initial_run, simple_phys
+use cam_control_mod,        only: initial_run
+use cam_physics_control,    only: simple_phys
```

Comment:
i think this should come from `cam_control_mod` (as `cam_physics_control` won't come in until my history PR) 

::grimace::

Reply:
Good catch!  Definitely missed that line during cleanup.